docs/knowledgebase/advanced/try-runtime.md

```diff
@@ -0,0 +1,184 @@
+---
+title: try-runtime
+---
+
+The `try-runtime` tool is built to query a snapshot of runtime storage, using an in-memory-externalities to store state. In this way, 
```

Comment:
Is there an explainer on externalities and the interface between runtime <-> client? Would be great to link with `in-memory-externalities`

Reply:
@emostov could you point me to some example of `in-memory-externalities`? 🙏🏻

Reply:
I think it actually just reffers to `TestExternalities` - in memory reffering to the fact that in just exists in a hash map and is not rewritten to disk. I guess we can link just to https://substrate.dev/rustdocs/latest/sp_state_machine/struct.TestExternalities.html (but not worth it if we already link imo bc just more maintenance)